hera_qm/utils.py

```diff
@@ -138,3 +140,79 @@ def generate_fullpol_file_list(files, pol_list):
                 file_list.append(jd_list)
 
     return file_list
+
+
+def metrics2mc(filename, ftype):
+    """ Read in file containing quality metrics and stuff into a dictionary which
```

Comment:
In the commit that's on its way I've added to the docstring explaining how to add a metric. But the long-term solution will be to add a memo and link in the readme once we undergo other reorganizations (a la issue #36).